src/test/files/OptimizeSQLQueriesWithLimit.java

```diff
@@ -0,0 +1,18 @@
+class OptimizeSQLQueriesWithLimit {
+
+    public void literalSQLrequest() {
+        dummyCall("SELECT user FROM myTable"); // Noncompliant {{Optimize Database SQL Queries (Clause LIMIT)}}
+        dummyCall("SELECT user FROM myTable LIMIT 50"); // Compliant
+    }
+
+    @Query("select t from Todo t where t.status != 'COMPLETED'") // Noncompliant {{Optimize Database SQL Queries (Clause LIMIT)}}
```

Comment:
In the JavaScript plugin, I added the WHERE keyword in the list of limiting keywords so as not to be too strict either. We could consider that this query already performs a stricter selection than all the data in the table. I don't know what your opinion is on this? maybe it lacks measure too :p

Reply:
@utarwyn update done